lcls_live/datamaps/klystron.py

```diff
@@ -266,9 +265,13 @@ def klystron_pvinfo(sector, station, beamcode=1):
         description += ' for special feedback'
         base =  f'KLYS:LI{sector}:{station}1'     # Normal base
         phase = f'ACCL:LI24:{station}00:KLY_PDES' # Readback
+        if beamcode == 2:
```

Comment:
You should consider implementing beamcode support for L0A, L0B, L1S, and L1X as well:
For Beamcode 1 amplitudes (aka enlds):
ACCL:IN20:300:L0A_AACT_DS0
ACCL:IN20:400:L0B_AACT_DS0
ACCL:LI21:1:L1S_AACT_DS0
ACCL:LI21:180:L1X_AACT_DS0

For Beamcode 2 amplitudes (aka enlds):
ACCL:IN20:300:L0A_AACT_DS1
ACCL:IN20:400:L0B_AACT_DS1
ACCL:LI21:1:L1S_AACT_DS1
ACCL:LI21:180:L1X_AACT_DS1

For Beamcode 1 phases:
ACCL:IN20:300:L0A_PACT_DS0
ACCL:IN20:400:L0B_PACT_DS0
ACCL:LI21:1:L1S_PACT_DS0
ACCL:LI21:180:L1X_PACT_DS0

For Beamcode 2 phases:
ACCL:IN20:300:L0A_PACT_DS1
ACCL:IN20:400:L0B_PACT_DS1
ACCL:LI21:1:L1S_PACT_DS1
ACCL:LI21:180:L1X_PACT_DS1

Reply:
`ACCL:IN20:400:L0B_AACT_DS` is missing from the archiver

Reply:
@mattgibbs Okay, I added all of these